pkg/asset/machines/aws/machines.go

```diff
@@ -81,7 +83,15 @@ func provider(clusterID, clusterName string, platform *aws.Platform, mpool *aws.
 			APIVersion: "awsproviderconfig.k8s.io/v1alpha1",
 			Kind:       "AWSMachineProviderConfig",
 		},
-		InstanceType:       mpool.InstanceType,
+		InstanceType: mpool.InstanceType,
+		BlockDevices: []awsprovider.BlockDeviceMappingSpec{
+			{
+				EBS: &awsprovider.EBSBlockDeviceSpec{
+					VolumeType: volumeType,
+					VolumeSize: &volumeSize,
```

Comment:
Looks like you left out [IOPS][1]?  That won't matter for the gp2 default, but would matter if we were passing through values configured via an `install-config.yaml` for io1.

[1]: https://github.com/openshift/installer/blob/v0.10.0/pkg/types/aws/machinepool.go#L51-L52